README.md

```diff
@@ -1 +1,70 @@
-# angular-routing-testing-readme
\ No newline at end of file
+# Testing Routing
+
+## Overview
+
+Now that we've got our routes setup, it's crucial that when we make changes to our application that we don't break any existing route - that's where tests come in!
+
+## Objectives
+
+- Describe routing testing
+- Write a test for routes and expect API calls/templates/Controllers
+
+## Instructions
+
+When we test our routes/states we should check that the URL is as we expect, that it's using the correct template and fetching the correct data that it needs.
```

Comment:
This is a short blog post that might be helpful -
https://robots.thoughtbot.com/how-we-test-rails-applications#controller-specs
Also the guides are good as well -
http://guides.rubyonrails.org/testing.html#functional-tests-for-your-controllers
Let me know if that's helpful!

On Tue, Mar 22, 2016 at 12:37 AM, Todd Motto <notifications@github.com>
wrote:

> In README.md
> <https://github.com/learn-co-curriculum/angular-routing-testing-readme/pull/1#discussion_r56936538>
> :
>
> > \ No newline at end of file
> > +# Testing Routing
> > +
> > +## Overview
> > +
> > +Now that we've got our routes setup, it's crucial that when we make changes to our application that we don't break any existing route - that's where tests come in!
> > +
> > +## Objectives
> > +
> > +- Describe routing testing
> > +- Write a test for routes and expect API calls/templates/Controllers
> > +
> > +## Instructions
> > +
> > +When we test our routes/states we should check that the URL is as we expect, that it's using the correct template and fetching the correct data that it needs.
>
> I'm not familiar with controller tests in Rails - is there a good article
> on this?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/learn-co-curriculum/angular-routing-testing-readme/pull/1/files/b9cd2fe31d51407ad0837bf65f78b80399ddfe2d#r56936538>
>



-- 


     Ian Candy
     Web Instructor